test/plugin/scenarios/jetty-scenario/jettyclient-scenario/src/main/java/org/apache/skywalking/apm/testcase/jettyclient/controller/CaseController.java

```diff
@@ -45,6 +46,8 @@ public void init() throws Exception {
     @ResponseBody
     public String jettyClientScenario() throws Exception {
         client.newRequest("http://" + jettyServerHost + ":18080/jettyserver-case/case/receiveContext-0").send();
+        Response.CompleteListener listener = result -> { };
```

Comment:
if you do something in listener such as query mysql, the trace context can not been got because your plugin just put span in request attribute

Reply:
@yangyulely I think the suggestion is, you should have improvements for the listener callback, such as a wrapper and a local span created for further processes in the listener.

Reply:
I've try to a wrapper for listener, and reset the argument to the new wrapper, but it looks like the argument was not that setted. could you share some experience or suggestion for it?

Reply:
You need to declare argument override in the instrument, please read the doc.